File: PlatziCodingChallenge_Dia18/MemoryParte1.py
#Memory (parte 1) en Python
from random import choice

def randomChars(chars, number):
    charsList = []
    for i in range(0, number):
        charsList.append(choice(chars))
    return charsList

caracteres = ['"', '#', '$', '%', '(', ')', '/', '&']
print(randomChars(caracteres, 4))

File: PlatziCodingChallenge_Dia18/MemoryParte1.js
//Memory (parte 1) en JavaScript

//Funcion que devuelve un elemento aleatorio de un array
function randomChar(chars) {
    // const chars = ['"', '#', '$', '%', '(', ')', '/', '&']
    return chars[Math.floor(Math.random()*chars.length)]
}

//Impresion de caracter aleatorio
console.log(randomChar(['"', '#', '$', '%', '(', ')', '/', '&']))